_docs/using-beaker/index.md

```diff
@@ -1,28 +1,224 @@
+  <figcaption class="thick">Click "Import files"</figcaption>
+</figure>
+
+</section>
+
+<section markdown="1">
+
+## Publishing files and changes
+
+When you're ready to share the files, click "Publish."
+
+Even if somebody has the URL of your site, they won't be able to access the
+files until you "publish" them. That's so you can review your changes first.
+
+<figure>
+  <img class="screenshot" src="/img/docs/tour-new-site-publish3.png"/>
+  <figcaption class="thick">
+    When you're ready to share, click "Publish"
+  </figcaption>
+</figure>
+
+</section>
+
+<section markdown="1">
+
+## How to share a site
+
+When you're ready to share, you just need to copy and paste the URL.
+
+<figure>
+  <img class="screenshot" src="/img/docs/tour-share-site1.png"/>
+  <figcaption class="thick">Click "Share site"</figcaption>
+</figure>
+
+<figure>
+  <img class="screenshot" src="/img/docs/tour-share-site2.png"/>
+  <figcaption class="thick">Copy the URL</figcaption>
+</figure>
+
+</section>
+
+<section markdown="1">
+
+## How to manage your site
+
+Beaker keeps all the sites you create in the "Library." You can find a link to
+it in your main menu.
+
+<figure>
+  <img class="screenshot" src="/img/docs/tour-library1.jpg"/>
+  <figcaption class="thick">Click "Library" in the browser dropdown menu</figcaption>
+</figure>
+
+</section>
+
+<section markdown="1">
+
+## Using the sidebar
+
+If you're visiting a peer-to-peer site, you can quickly access more information
+by using the sidebar control.
+
+<figure>
+  <img class="screenshot" src="/img/docs/tour-sidebar1.png"/>
+  <figcaption class="thick">Click the sidebar icon in the navbar</figcaption>
 </figure>
 
-You can see how many peers are hosting a site’s files. When you navigate to a site, you’ll automatically share the files you download for a short period. You can [save the site to your library](/docs/using-beaker/saving-sites.html) if you want to rehost it permanently.
+</section>
+
+<section markdown="1">
+
+## Live reloading
+
+As the author of a site, or just as a remote recipient, live-reloading makes it
+easy to stay abreast of changes while you work. Turn it on to have the page
+automatically update every time you make a change.
+
+<figure>
+  <img class="screenshot" src="/img/docs/tour-livereloading.jpg"/>
+  <figcaption class="thick">The yellow bolt will indicate it's on</figcaption>
+</figure>
+
+</section>
+
+<section markdown="1">
+
+## Sync for offline
+
+As the visitor to a site, you may want to save a site for offline reading.
+Turn on "Sync for offline" to make sure you get the latest updates when online,
+and then can access the files when offline.
+
+<figure>
+  <img class="screenshot" src="/img/docs/tour-sync-for-offline2.png"/>
+  <figcaption class="thick">Select "Sync for offline"</figcaption>
+</figure>
+
+</section>
+
+<section markdown="1">
+
+## Forking a site
+
+Sometimes, you want to create a new copy of a site that you control. This is
```

Comment:
Is this supposed to say "a site that you don't control?"

Also, I think something like "a website that you aren't the author of" is more clear